xbmc/cores/VideoPlayer/DVDInputStreams/DVDInputStreamBluray.cpp

```diff
@@ -467,6 +529,15 @@ void CDVDInputStreamBluray::Close()
   }
   m_bd = NULL;
   m_title = NULL;
+  if (m_pstream != nullptr)
```

Comment:
`m_pstream` and `m_rootPath` will never be null here, their content might be that's not relevant so this can be simplified as
```
m_pstream.reset();
m_rootPath.reset();
```
as `Close()` will be called by the destructor when calling `reset()`